ci/prow/e2e_prebuilt_kernel_module_two_nodes

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
Some the comments from [e2e_prebuilt_kernel_module](https://github.com/kubernetes-sigs/kernel-module-management/pull/159/files#diff-9c62c51855541d7fc09bf9d7021b7f647169e182aac4396e55c9e58f1bd86eca)` applies here as well.

Here I will just add additional comments if there are such.